tests/testthat/test-length.R

```diff
@@ -221,3 +221,7 @@ test_that("xportr_length: error when metadata is not set", {
     regexp = "Metadata must be set with `metadata` or `xportr_metadata\\(\\)`"
   )
 })
+
+test_that("xportr_length: Gets warning when metadata has multiple rows with same variable", {
+  multiple_vars_in_spec_helper(xportr_length)
```

Comment:
Can we add some expectations in this test and test-order.R, test-type.R?

Reply:
Done! I did it this way to avoid duplicate code on all tests and following `testthat` best practices.